posts/2022-01-14-applying-domain-driven-design-to-blockchains.md

```diff
@@ -0,0 +1,192 @@
+---
+title: 'Applying Domain Driven Design to Blockchains'
+description: 'This is the post about 4 DDD Patterns for your Next Distributed Ledger or Permissioned Private Blockchain created on Vendia Share data platform'
+date: '2022-01-14'
+thumbnail: 'https://user-images.githubusercontent.com/96793170/151107275-fa416768-f940-407c-9e03-5b264ea61e10.jpg'
+authors:
+  - Vikrant Kahlir
+---
+
+# Applying Domain Driven Design to Blockchains
+
+Vendia Share is a platform that you can use to quickly build real-time data sharing applications which we call Universal Applications (or "Unis" for short). 
+
+Data models power Universal Applications.  The schema-first approach makes it easy to design and create Unis. In fact, design and create are almost indistinguishable steps after you have a ready-to-deploy schema. Vendia Share provides you a GraphQL API based on your data model that you and your partners can use to read and write data into Vendia Share. It also generates an immutable log of all transactions. Because a data model is a critical piece needed to create a Uni, designing the right data model for your application is an essential step to use Vendia Share. 
```

Comment:
```suggestion
Data models power Universal Applications.  The schema-first approach makes it easy to design and create Unis. In fact, design and create are almost indistinguishable steps after you have a ready-to-deploy schema. Vendia Share provides a GraphQL API based on your data model. You and your partners can use that API to read data from and write data to Vendia Share. Because a data model is a critical piece needed to create a Uni, designing the right data model for your application is an essential step to using Vendia Share. 
```